src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -326,21 +328,20 @@ The `description` property describes the purpose of your app.
 
 The `screenshots` property is an array of image objects representing your app
 in common usage scenarios. Each object must include the `src`, a `sizes`
-property, and the `type` of image.
+property, and the `type` of image.  The `form_factor` property is optional.
+It can be set either to `"wide"` for screenshots applicable to wide screens
+only or `"narrow"` for narrow screenshots. You should only use it when the
+layout varies by screen size.
 
 In Chrome, the image must respond to certain criteria:
 
 * Width and height must be at least 320px and at most 3840px.
 * The maximum dimension can't be more than 2.3 times as long as the minimum
   dimension.
-* All screenshots must have the same aspect ratio.
+* All screenshots matching the appropriate form factor must have the same
+  aspect ratio.
 * Only JPEG and PNG image formats are supported.
 
-{% Aside 'gotchas' %}
```

Comment:
I'm guessing we'll wait to publish this PR until the richer install ui is published for Desktop?

Reply:
Indeed.